lib/graphql/schema.ts

```diff
@@ -16,6 +16,7 @@ export default `
     updateApplicant(input: UpdateApplicantInput!): UpdateApplicantResult!
     createEmployee(input: CreateEmployeeInput!): CreateEmployeeResult!
     updateEmployee(input: UpdateEmployeeInput!): UpdateEmployeeResult!
+    deleteEmployee(input: DeleteEmployeeInput!): DeleteEmployeeResult!
```

Comment:
Since there is only one input parameter (ID) we can just pass it in directly, no need for putting it in an object:
```suggestion
    deleteEmployee(id: ID!): DeleteEmployeeResult!
```
and the `DeleteEmployeeInput` type should be removed from `lib/employees/schema.ts`